controller/config/samples/kontroler_v1alpha1_dag_event_long.yaml

```
@@ -0,0 +1,191 @@
+      command: ["sh", "-c"]
+      args:
+        [
+          "if [ $((RANDOM%2)) -eq 0 ]; then echo 'Hello, World!'; else exit 1; fi",
+        ]
+      image: "alpine:latest"
+      runAfter: ["random-b"]
+      backoff:
+        limit: 3
+      conditional:
+        enabled: true
+        retryCodes: [1]
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)